java/src/jmri/jmrit/logix/WarrantTableModel.java

```diff
@@ -433,9 +433,9 @@ public Object getValueAt(int row, int col) {
 
     @Override
     public void setValueAt(Object value, int row, int col) {
-        if (log.isDebugEnabled())
-            log.debug("setValueAt: row= " + row + ", column= " + col
-                    + ", value= " + (value==null ? value : (value.toString()==null ? value.getClass().getName() :value.toString())));
+//        if (log.isDebugEnabled())
```

Comment:
If you want to keep this available, but not have it normally logged in debug mode, you can use ```log.trace(...)``` instead of commenting it out.